docs/vsphere-golden-image.md

```diff
@@ -158,13 +160,24 @@ where `<path/to/unattend.xml>` is the path to the customized answer file.
 Note: There is [a limit](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation#limits-on-how-many-times-you-can-run-sysprep)
 on how many times you can run the `sysprep` command.
 
-## 7. Set up the virtual machine golden image
+## 7. Set up the virtual machine template
 
 Once the `sysprep` command completes the Windows virtual machine will power off. 
 
-You **must not** use this Windows virtual machine anymore. 
+You **must not** use or power on this virtual machine anymore.
+
+Note: In case, you need to update or change the configuration of the golden image, make the changes in the 
+original Windows virtual machine (before running the `sysprep` command), clone it and then execute `sysprep` command.
```

Comment:
I suggest leaving this out. It is a bit confusing. I know this is in the existing docs and we are trying to cover all cases but lets leave it out for simplicity.